spec/unit/swagger/swagger_spec.rb

```diff
@@ -194,6 +194,78 @@
         expect(get_findPetsByStatus_operation.operation_id).to eq('findPetsByStatus')
       end
     end
+
+    context 'extensions in path object' do
+      let(:content) do
+        <<~YAML
+          ---
+          swagger: "2.0"
+          info:
+            title: "#{title}"
+            version: "1.0.0"
+          paths:
+            /pet:
+              get:
+                operationId: "getPet"
+                responses:
+                  200:
+                    description: "successful operation"
+              x-internal-id: 383883
+        YAML
+      end
+      it 'parsed as not empty' do
+        expect(subject.operations).not_to be_empty
+      end
+
+      it 'parsed single operation' do
+        expect(subject.operations.size).to eq(1)
+      end
+
+      it 'parsed operation path /pet' do
+        expect(subject.operations[0].path).to eq('/pet')
+      end
+      it 'parsed operation method get' do
+        expect(subject.operations[0].verb).to eq('get')
+      end
+    end
+
+    context 'parameters in path item object' do
```

Comment:
Would it be a good idea to test that there are parameters?

Reply:
`parameters` fields are not parsed, they are filtered out. The context says there are parameters. If there are no parameters fields in the `content`, the test is useless.

To test that there are `parameters`, you could parse `content` and make sure parameters are there. but I think it is so obvious because there is a specific context for that case

Reply:
In other words, if there is a context for some var `A` to be, let's say, `2`, I think it is so obvious that there is no need to check that `A` is indeed `2`

Reply:
:+1: